addon/serializers/github-repository.js

```diff
@@ -7,6 +7,13 @@ export default GithubSerializer.extend({
       fullName: hash.full_name,
       name: hash.name,
       description: hash.description,
+      url: hash.html_url,
+      language: hash.language,
+      isFork: hash.fork,
+      isPrivate: hash.private,
```

Comment:
I've do the choice to break the naming convention from github, but it's logic for me : 
* url : url means, for github API, "The API endpoint to get the repository" and for me, ember-data-github must be an abstraction between the API and the model, so when i want url from a repo, i don't want the URL of API.
* language : Clear.
* isFork : "is" keywork is to see the type easier. In this way, "fork" may be a reserved keyword can load async the list of forks gettable with fork_url.
* isPrivate : may be renamed in private, but, if we keep "isFork", it's logic to keep "isPrivate".

It's your repo, and your rules. It's just my 2 cents ;)